docs/rss.md

```diff
@@ -0,0 +1,19 @@
+# RSS Feed
+
+RSS Feed (“Rich Site Summary” or “Really Simple Syndication”) is type of web feed that provides users and applications with regular updates from a given website. The RSS feed is a structured XML code document.
+
+By default RSS Feed is enabled. If you wish to disable it, set the `rss` configuration property to `false`
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cms": {
+					"rss": false
```

Comment:
I'd probably call this `feeds` rather than `rss` (and just broadly call it "Feeds") as it's a bit broader than specifically just RSS.

Reply:
Ok, changed to feeds.